ui/src/identity.ts

```diff
@@ -19,6 +19,7 @@ export interface Identity {
   };
   registered?: string;
   avatarFallback: Avatar;
+  shareableEntityIdentifier?: string;
```

Comment:
I'd also consider pushing out the use of `string` everywhere and introducing types that have a better structure and/or allow us to disambiguate them from other `string`ly typed objects.

Reply:
agreed

Reply:
This might get a little awkward for components that render `string`s though, like `Title` or `Text`

Reply:
Oh ya, sure, They're literally strings. No argument there :)